synapse/lib/ast.py

```diff
@@ -2655,7 +2663,8 @@ async def getPropAndValu(self, runt, path):
 
             prop = node.form.props.get(name)
             if prop is None:
-                raise s_exc.NoSuchProp(name=name, form=node.form.name)
+                mesg = f'No property named {name}.'
```

Comment:
For this to happen, syntactically, we'd have to have a node which had props in its propbag which are not present in the data model. That's a pretty big inconsistency..